packages/rollup-contracts/contracts/CanonicalTransactionChain.sol

```diff
@@ -0,0 +1,115 @@
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+/* Internal Imports */
+import {DataTypes as dt} from "./DataTypes.sol";
+import {RollupMerkleUtils} from "./RollupMerkleUtils.sol";
+import {L1ToL2TransactionQueue} from "./L1ToL2TransactionQueue.sol";
+
+contract CanonicalTransactionChain {
+  address public sequencer;
+  uint public sequencerLivenessAssumption;
```

Comment:
I know we've used this term in passing before, but a name more related to queueing might be good here as "liveness" is overloaded in crypto generally.  `forceInclusionPeriod`?  Idk hahaha